pandas/io/json/normalize.py

```diff
@@ -132,6 +147,17 @@ def json_normalize(data, record_path=None, meta=None,
 
         .. versionadded:: 0.20.0
 
+    max_level : integer, default None
+        max number of levels(depth of dict) to normalize.
+        if None, normalizes all levels.
+
+        .. versionadded:: 0.25.0
+
+    ignore_keys : list, keys to ignore, default []
```

Comment:
see my comments above

Reply:
This is a mistake. Default is `None`. @WillAyd mentioned in one of the old comments